src/async_impl/decoder.rs

```diff
@@ -604,6 +604,7 @@ impl Accepts {
     }
 }
 
+#[allow(clippy::derivable_impls)]
```

Comment:
Could we leave these allows out? We don't run clippy in CI (particularly because it has false positives like these, and I find the allows distracting...)